downstream/modules/automation-hub/proc-install-ha-hub-selinux.adoc

```diff
@@ -1,7 +1,7 @@
 
 = Install a high availability (HA) deployment of {HubName} on SELinux
 
-To set up a high availability (HA) deployment of {HubName} on SELinux, create two mount points for `/var/lib/pulp` and `/var/lib/pulp/pulpcore_static`, then assign the appropriate SELinux contexts to each. You must add the context for `/var/lib/pulp` and run the {PlatformNameShort} installer before adding the context for `/var/lib/pulp/pulpcore_static`.
+To set up a high availability (HA) deployment of {HubName} on SELinux, create two mount points for `/var/lib/pulp` and `/var/lib/pulp/pulpcore_static`, then assign the appropriate SELinux contexts to each. You must add the context for `/var/lib/pulp/pulpcore_static` and run the Ansible Automation Platform installer before adding the context for `/var/lib/pulp`.
```

Comment:
You probably have a reason to replace the platform attribute with the hard coded "Ansible Automation Platform". Just wondering if we should use the attribute?

Reply:
Thanks, Jameria! That was just an oversight on my part. I've edited the text to include the attribute.

Reply:
No problem, Lizzi! Glad to help. LGTM! Squashing and merging in just a few...